features/step_definitions/follow_steps.rb

```diff
@@ -0,0 +1,13 @@
+When(/^I visit a user's profile$/) do
+  user2 = FactoryGirl.create(:user, id: 2, email: "newuser2@sb.com")
```

Comment:
Any reason you're creating a user with a specific id here? it doesn't seem to be used later in the step.